Procurement/View/VisibilityConverter.cs

```diff
@@ -30,7 +30,7 @@ public object Convert(object value, Type targetType, object parameter, System.Gl
 
                     if (enumerable != null)
                     {
-                        result = enumerable.Cast<object>().Any() ? Visibility.Hidden : Visibility.Visible;
+                        result = enumerable.Cast<object>().Any() ? Visibility.Visible : Visibility.Hidden;
```

Comment:
This is inverting the logic that is already in place - is this intended? May have repercussions where it is already in use. 

I tend to favour Collapsed compared to Hidden as I don't want the space reserved for the element when it's not visible.